experimentation/tools/achievements/README.md

```diff
@@ -0,0 +1,26 @@
+# Achievements script
+This is a script that can be used to generate an "achievements" file for
+Sorald, detailing the pull requests performed with it. It's essentally
```

Comment:
English is not my mother tongue, thus I'm sorry in advance if I'm wrong:
"essentally" -> "essentially"

Reply:
That's definitely a typo, fixed!